templates/html/extsearch.js

```diff
@@ -78,16 +78,60 @@ function searchFor(query,page,count) {
       $.each(data.items, function(i,item){
         let prefix = tagMap[item.tag];
         if (prefix) prefix+='/'; else prefix='';
+
+        icon = "asterisk";
+
+        if (item.type == "dir")
```

Comment:
Suggest to use a `map` instead of `if..else if..else if`, i.e.

```js
const myMap = {
    "key 1" : "value 1", 
    "key 2" : "value 2", 
    ...
    "key n" : "value n"
};
```